src/Ledger/RolloverLedger/RolloverLedgerContainer.js

```diff
@@ -107,7 +111,45 @@ export const RolloverLedgerContainer = ({ resources, mutator, match, history, lo
     [ledger, rollover, showCallout],
   );
 
+  const checkIsRolloverAlreadyExist = useCallback(async (currentRollover) => {
+    const { fromFiscalYearId: fromFY, toFiscalYearId: toFY } = currentRollover;
+    const query = `ledgerId==${ledgerId} and rolloverType=="${LEDGER_ROLLOVER_TYPES.commit}" and fromFiscalYearId==${fromFY} and toFiscalYearId==${toFY}`;
+    const { totalRecords } = await ky.get(LEDGER_ROLLOVER_API, { searchParams: { query } }).json();
+
+    if (totalRecords) {
+      const { name: ledgerName } = ledger || {};
+      const fromFYcode = fiscalYears?.find(({ id }) => id === fromFY)?.code;
+      const toFYcode = fiscalYears?.find(({ id }) => id === toFY)?.code;
+
+      const message = intl.formatMessage(
+        {
+          id: 'ui-finance.ledger.rollover.error.conflict',
+        },
+        {
+          ledgerName,
+          fromFYcode,
+          toFYcode,
+        },
+      );
+
+      showCallout({
+        message,
+        type: 'error',
+      });
+
+      return true;
+    }
+
+    return false;
+  }, [fiscalYears, intl, ky, ledger, ledgerId, showCallout]);
+
   const showConfirmation = useCallback(async (rolloverValues) => {
+    const isRolloverExist = await checkIsRolloverAlreadyExist(rolloverValues);
```

Comment:
If the request fails, the error is not processed, therefore the user does not see any error message.